instrumentation-api-incubator/build.gradle.kts

```diff
@@ -42,7 +42,7 @@ tasks {
   }
 
   val testStableSemconv by registering(Test::class) {
-    jvmArgs("-Dotel.semconv-stability.opt-in=database")
+    jvmArgs("-Dotel.semconv-stability.opt-in=database,http")
```

Comment:
stable http semconv is the default since 2.0 (and we intentionally dropped support for the old http semconv in 2.0)

if users need the old http semconv, they'll need to use 1.x

Reply:
ah, can you summarize in the PR description what metrics and/or attributes you want to add? thanks!

Reply:
I have updated it in the PR details.